crux-http-server/src/crux/http_server.clj

```diff
@@ -416,35 +557,18 @@
      (log/info "HTTP server started on port: " server-port)
      (->HTTPServer server options))))
 
-(defn- edn->html [edn]
-  (cond
-    (map? edn) (into [:dl]
-                     (mapcat
-                      (fn [[k v]]
-                        [[:dt (edn->html k)]
-                         [:dd (edn->html v)]])
-                      edn))
-    (sequential? edn) (into [:ol] (map (fn [v] [:li (edn->html v)]) edn))
-    (set? edn) (into [:ul] (map (fn [v] [:li (edn->html v)]) edn))
-    :else (str edn)))
-
-(defn- entity->html [edn]
-  (str
-   (html
-    (edn->html edn))))
-
 (defn- html-encoder [_]
   (reify
     mfc/EncodeToBytes
     (encode-to-bytes [_ data charset]
       (.getBytes
-       ^String (entity->html data)
+       ^String (pr-str data)
```

Comment:
I know we're not using this now, because we're manually specifying formats for each handler, but `pr-str` doesn't seem like a reasonable default - shall we `(throw (UnsupportedOperationException.))` instead, to be really clear that we don't expect to get here?

Reply:
It still gets here, we're still using html-encoder (to encode the HTML strings returned from the respective handlers)